### ISSUE TITLE
SSL Support?

### ISSUE THREAD

**Issue by arch1v1st** [arcoirislabs/cordova-plugin-mqtt#23]
Hi, 

What are the blockers to support SSL encryption for all communications?

**Comment by nmaas87** [comment]
That would be really good to use 👍

**Comment by pavankumarkatakam** [comment]
yeah, this is useful for working with AWS IOT. The underlying library (Paho android ) has already support for  SSL and TLS https://github.com/eclipse/paho.mqtt.android#features

**Comment by arcoirislabs** [comment]
@pavankumarkatakam @nmaas87 We are planning the SSL support in the forthcoming release

**Comment by bkrajendra** [comment]
SSL working fine for me with android see this #37
